cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -332,92 +82,60 @@ def import_status_handler(request, course_key_string, filename=None):
 @ensure_valid_course_key
 def export_handler(request, course_key_string):
     """
-    The restful handler for exporting a course.
+    The restful handler for the export page.
 
     GET
         html: return html page for import page
-        application/x-tgz: return tar.gz file containing exported course
-        json: not supported
+    """
+    error = request.GET.get('error', None)
+    error_message = request.GET.get('error_message', None)
+    failed_module = request.GET.get('failed_module', None)
+    unit = request.GET.get('unit', None)
+    parent = request.GET.get('parent', None)
 
-    Note that there are 2 ways to request the tar.gz file. The request header can specify
-    application/x-tgz via HTTP_ACCEPT, or a query parameter can be used (?_accept=application/x-tgz).
+    if parent:
+        try:
+            parent = CourseKey.from_string(parent)
+        except InvalidKeyError:
+            parent = None
```

Comment:
I would expect this to trigger some kind of error response here, rather than setting `parent = None` and moving on. If someone hits this endpoint and provides a `parent` parameter, but there is some error in transforming that into a `CourseKey`, then this code will silently drop that parameter and move on -- a very difficult situation to debug! Can you make it display an error in the HTML page if that is the case? (If this was a JSON API, I'd suggest that you immediately return a 400 status code response, but you've changed this to only output HTML.)